packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -8,23 +9,54 @@ import { useMenuTriggerState } from '@react-stately/menu';
 import { useOverlayPosition } from '@react-aria/overlays';
 
 interface DropdownProps {
+  /**
+   * Alignment of the menu relative to the trigger.
+   *
+   * @default 'start'
+   */
+  align?: 'end' | 'start';
```

Comment:
I'm surprise there is no `center`.

Reply:
we can add it for sure!

Reply:
Looking at react-aria api for the overlay, looks like 'center' isn't supported, rather if you want to center something `align` should be undefined. I think we can protect against this, and add center and adjust the initial placement accordingly.